web/README.md

```diff
@@ -22,3 +24,13 @@ To start working with WEB OTE it is required to set up it locally. To do it choo
 # Get Started
 Please refer [Getting Started](GETTING_STARTED.md) page for the basic usage. 
 
+# UI Examples
+Face Detection Page:
+![face-detection](.images/face-detection.png)
+
+Fine Tune Dialog:
+![fine-tube](.images/fine-tune.png)
+
+Assets Page:
+![assets](.images/assets.png)
```

Comment:
BTW, Images resolution is 1848x1006.
Consider adapting its size for GitHub's README.md layout:
- hard to see details on FHD monitors
- avoid resize artifacts.

"Live" demo: https://github.com/openvinotoolkit/training_extensions/tree/add-images-to-readme/web